compiler/dyno/include/chpl/uast/Function.h

```diff
@@ -220,6 +220,11 @@ class Function final : public NamedDecl {
   bool isPrimaryMethod() const { return primaryMethod_; }
   bool isParenless() const { return parenless_; }
 
+  bool isAnonymous() const {
+    auto ret = (kind() == LAMBDA || name() == "proc");
```

Comment:
Just to confirm: no function in Chapel code can ever be called `proc` since `proc` is a keyword, so this comparison is fine?

Reply:
Yeah, that's the idea behind this shortcut. We could also add different "Function::Kind" as well if we find this distasteful, I suppose. Or a `bool is Anonymous` field.